docs/Examples/Basic.md

```diff
@@ -156,9 +156,14 @@ Great, now we're all set. Just need to create our provider.
 ```swift
 // Tuck this away somewhere where it'll be visible to anyone who wants to use it
 var provider: MoyaProvider<MyService>!
-
-// Create this instance at app launch
-let provider = MoyaProvider(endpointClosure: endpointClosure)
+```
+At app launch, you then need to instantiate your provider.
+```swift
+provider = MoyaProvider(endpointClosure: endpointClosure)
+```
+If by any chance you don't have access to a network, or your API is not ready yet, you can always simulate it by stubbing your requests.  Just make sure you have some `sampleData` in your enum set to the response you're expecting.
```

Comment:
Can we describe a little more the `stubClosure` parameter? Just a little bit about few options of stubbing, the default argument etc.

Reply:
My rationale behind this addition was something along the lines of _"hey, just in case you're wondering you can stub your requests"_. So beginners tinkering with for the first time don't have to go looking for specific sections in the documentation.

Maybe going into more depth regarding stubbing, could be done, as @SD10 suggested, in a `Stubbing.md`. There we could explain the different options of stubbing, the default argument, etc. 

I can do that write up.